app/views/play/cinematic/CinematicLankBoss.js

```diff
@@ -132,6 +136,26 @@ export default class CinematicLankBoss {
         })
       ]))
     }
+
+    const text = getText(dialogNode)
+    const animation = getSpeakingAnimation(dialogNode)
+    if (text && animation) {
+      const textLength = getTextAnimationLength(dialogNode)
+      const speaker = getSpeaker(dialogNode)
+      commands.push(new SequentialCommands([
+        new Sleep(Math.min(100, textLength)),
+        new SyncFunction(() => {
+          this.playActionOnLank(speaker, animation)
+        })
+      ]))
+
+      commands.push(new SequentialCommands([
+        new Sleep(textLength),
```

Comment:
Do we need a min sleep length here?

Reply:
The min sleep problem should not present here as the bug is the action not playing.
If the textLength is < 100 then the action not playing shouldnt be a problem.

You've made me realize there are some other cases that could be interesting. Like if the textLength is exactly 100. Then the animation and animation end will try to play at the same time.

I don't want to over complicate too much on this first iteration and instead want to see how the designers use it.
We have another window to fix these issues if they are a major concern.